rotary_utils/repair.py

```diff
@@ -158,27 +158,50 @@ def __init__(self, linking_outdir: str, length_threshold: int = None):
         """
         self.linking_outdir = linking_outdir
         self.log_dir_base = os.path.join(linking_outdir, 'logs')
-        self.length_threshold = length_threshold
+        self._current_length_threshold = length_threshold
```

Comment:
@jmtsuji We create a new variable called `_current_length_threshold` that represents the current state of the length threshold. Note that this variable starts with an underscore `_.` This is best practice indicating that this attribute is private and should only be accessed internally by the class, not outside, once the object is created. In Java, you can label attributes as `private,` which will raise an error if they are called outside the class. Python doesn't have this.